File: 412-fizz-buzz/fizz-buzz.java
class Solution {
  public List<String> fizzBuzz(int n) {
        List<String> list=new ArrayList<>();
        while(n!=0){
            if(n%3==0 && n%5 == 0){
                list.add("FizzBuzz");
            }else if(n%3==0){
                list.add("Fizz");
            }else if(n%5==0){
                list.add("Buzz");
            }else{
                list.add(""+n);
            }
            n--;
        }
        Collections.reverse(list);
        return list;
    }
}

File: 412-fizz-buzz/fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    
    var dataArray = [];

    for(var i=1 ;i < n+1 ;i++){

        dataArray[i-1] = `${i}`;

        if((i%5 === 0 ) && (i% 3 === 0)){

            dataArray[i-1]="FizzBuzz";
        }
        else if( i % 5 === 0){

            dataArray[i-1]='Buzz';
        }
        else if(i%3 === 0){
            
            dataArray[i-1]='Fizz';
        }

        //else{
        //  dataArray[i-1]= `${i}`;
        //}
    }

    return dataArray;
};